src/main/scala/mesosphere/marathon/state/Group.scala

```diff
@@ -40,6 +40,8 @@ class Group(
   lazy val transitiveApps: Set[AppDefinition] = transitiveAppsById.values.toSet
   lazy val transitiveAppIds: Set[PathId] = transitiveAppsById.keySet
 
+  lazy val transitivePods: Set[PodDefinition] = transitivePodsById.values.toSet
```

Comment:
I think I understand the need for this. I'm wondering how it'll impact benchmarks

Reply:
It seems we can safely make it a `def`.